velox/exec/tests/PlanNodeSerdeTest.cpp

```diff
@@ -195,6 +195,17 @@ TEST_F(PlanNodeSerdeTest, groupId) {
   testSerde(plan);
 }
 
+TEST_F(PlanNodeSerdeTest, expand) {
+  auto plan = PlanBuilder()
+                  .values({data_})
+                  .expand(
+                      {{"c0", "c1", "c2", "0 as group_id_0"},
+                       {"c0", "c1", "null as c2", "1  as group_id_0"},
```

Comment:
PlanBuilder seems to pick column names from the first list of projections. Hence, it is redundant to specify these for all projections. Let's remove these from 2nd and subsequent projections and add a comment to PlanBuilder::expand that explains this behavior.